Permanent/ViewControllers/FileDetailsViewController.swift

```diff
@@ -438,6 +454,11 @@ extension FileDetailsViewController: UICollectionViewDelegateFlowLayout {
             } else {
                 return CGSize(width: UIScreen.main.bounds.width, height: UIScreen.main.bounds.width * 0.65)
             }
+        case .tags:
+            let cellHeight: CGFloat = 40
+            let countTags = stringCellDetails(cellType: currentCellType).components(separatedBy: ",").count
+            let cellHeightByTagsNumber :CGFloat = CGFloat(( countTags > 3 ) ? ( (cellHeight * 3) + 13 ) : ( cellHeight * 2 ))
```

Comment:
Can you please add a comment here describing the logic of calculating the height? I find it a bit confusing, and it might be helpful for future us to know what the intention was

Reply:
A short comment was added.